scrapers/Mylf.yml

```diff
@@ -13,6 +13,13 @@ xPathScrapers:
         selector: //div[contains(@class,"sceneDate")]/text()
         postProcess:
             - parseDate: 01/02/2006
+      Tags:
+        Name: 
+          selector: //div[@class="col-12 tags-container"]//a
+          postProcess:
+            - replace: 
+              - regex: "[^a-zA-Z0-9]+"
+                with: $1   
```

Comment:
The `$1 ` here doesnt serve a purpose. You havent used a matching group in the regex so that is equivalent to
`with:  ""` or `with:`. I would use the `with: ""` . Also if you can add a comment next to it eg `comment the  whole postProcess if you want to preserve the original tags` that would be great. ( i for example like to keep whitespaces or `/` with my tags)